.gitignore

```diff
@@ -2,14 +2,16 @@ package-lock.json
 node_modules
 dist/
 
-baseTrie.js
-checkpoint-interface.js
-index.js
-prioritizedTaskExecutor.js
-proof.js
-readStream.js
-secure-interface.js
-secure.js
-scratchReadStream.js
-trieNode.js
-util.js
+/baseTrie.js
+/checkpointTrie.js
+/db.js
+/index.js
+/karma-conf.js
+/prioritizedTaskExecutor.js
+/proof.js
+/readStream.js
+/scratch.js
+/scratchReadStream.js
+/secure.js
+/trieNode.js
+util/
```

Comment:
@holgerd77  For the `gitignore` As you can see, I added all the files that I saw in the build, but arent on github. I also removed ones that dont seem to exist anymore like `checkpoint-interface.js` I replaced with `/checkpointTrie.js` and `secure-interface.js` was nowhere to be found.

Let me know if any of this was incorrect

Reply:
This karma-conf shouldn't be included. The util directory should very likely also have a trailing slash, otherwise src/util probably also isn't included.

Reply:
Generally this should replicate the structure from the src directory.

Reply:
I'm not sure if adding these to the `gitignore` is worthwhile, since in the future typescript PR they'd have to be removed again (as the build system will change).

Reply:
@s1na This is not adding, this is just fixing, entries are already there. Not for eternity of course, but let's keep until we switch.

Reply:
Ah, sorry my bad, here it only shows 4 added lines, I didn't check the full diff